packages/babel-plugin-transform-object-rest-spread/test/fixtures/object-spread/assignment/expected.js

```diff
@@ -1,5 +1,5 @@
 var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
 
-z = _extends({ x }, y);
+z = _extends({}, { x }, y);
```

Comment:
Totally, but given the issue name, I don't get the feeling that that is what this issue is trying to resolve, which is why I asked.